impala/dbapi.py

```diff
@@ -192,7 +192,7 @@ def connect(host='localhost', port=21050, database=None, timeout=None,
         http_cookie_names = auth_cookie_names
     elif http_cookie_names is None:
         # Set default value as the list of HTTP cookie names used by Impala and Hive.
-        http_cookie_names = ['impala.auth', 'impala.session.id', 'hive.server2.auth']
+        http_cookie_names = ['KNOXSESSIONID', 'KNOX_BACKEND-HIVE', 'JSESSIONID', 'impala.auth', 'impala.session.id', 'hive.server2.auth']
```

Comment:
Move the three new cookie names to the end of array.

Reply:
Moved the new cookie names to the end of array.